proxy_multi_map.go

```diff
@@ -0,0 +1,515 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package hazelcast
+
+import (
+	"context"
+	"time"
+
+	"github.com/hazelcast/hazelcast-go-client/internal/proto"
+	"github.com/hazelcast/hazelcast-go-client/internal/proto/codec"
+	"github.com/hazelcast/hazelcast-go-client/internal/serialization"
+	"github.com/hazelcast/hazelcast-go-client/types"
+)
+
+const (
+//	maxIndexAttributes               = 255
+//	defaultLockID                    = 0
+//	lockIDKey          lockIDKeyType = "__hz_lock_id"
+//	leaseUnset                       = -1
+)
+
+//type lockID int64
+//type lockIDKeyType string
+
```

Comment:
Forgotten commented lines

Reply:
https://github.com/hazelcast/hazelcast-go-client/pull/689#discussion_r744799439

actually I wanted hear your thoughts for these, but some of your comments already addressed it. I am removing these